### ISSUE TITLE
Display tree with any node as root

### ISSUE THREAD

**Issue by KopolJunam** [denkbar/djigger#86]
When the interesting method is on the 50th level of the call stack it would be convenient if one could create a new tree view with the node in question as root. E.g. in context menu as "Make this node root".

**Comment by dcransac** [comment]
True, we've wanted to declare tree "entry" (root) and tree "exit" (leaf) points for a while to allow the user to define where they want to start a tree and where a newly created, artificial tree should start and end.

Interesting to know some else wants that. We'll take this in account.
